web/test/tests/almafix/LocalIndexSetup.java

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2017 Fabian Steeg, hbz. Licensed under the EPL 2.0 */
+
+package tests.almafix;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+import controllers.resources.LocalIndex;
+import controllers.resources.Search;
+
+/**
+ * Setup for the alma fix search tests. Creates a local ES index with test data.
+ *
+ * @author Fabian Steeg (fsteeg)
+ */
+@SuppressWarnings("javadoc")
+public abstract class LocalIndexSetup {
+
+    private static LocalIndex index;
+    private static String TEST_CONFIG =
```

Comment:
This could be `final`, right? If not, as above, it should  be called `testConfig`.

Reply:
right. fixed that.